src/github/client/key-locator.test.ts

```diff
@@ -20,7 +20,7 @@ describe("key-locator", () => {
 			installationId: 1
 		});
 
-		const privateKey = await keyLocator(gitHubServerApp.id);
+		const privateKey = await keyLocator(gitHubServerApp.appId);
```

Comment:
Again, this needs to be the unique id, not the potentially repeated appId.

Reply:
Reverted and fixed. Closing this thread.